src/custom-forms/use-custom-forms-prefetch.js

```diff
@@ -1,12 +1,25 @@
+import { useAlert } from '@dhis2/app-runtime'
+import i18n from '@dhis2/d2-i18n'
 import { useQueryClient } from '@tanstack/react-query'
 import keys from './query-key-factory.js'
 import useCustomForms from './use-custom-forms.js'
 
+function getAlertSettings() {
+    return { warning: true }
+}
+
 const useCustomFormsPrefetch = () => {
+    const { show: showAlert } = useAlert((message) => message, getAlertSettings)
     const queryClient = useQueryClient()
-    const { isSuccess, data } = useCustomForms()
+    const { isSuccess, isError, error, data } = useCustomForms()
 
-    if (isSuccess) {
+    if (isError) {
+        showAlert(
+            i18n.t("Custom forms couldn't be saved offline. {{error}}", {
```

Comment:
I'm a bit confused about this message? Isn't this about custom forms failing to load? Not saved?

Reply:
Yeah, good point @Birkbjo, this could be misinterpreted. I think `Custom forms couldn't be made available offline.` is clearer.

Reply:
I updated the value. You OK with this @Birkbjo?